packages/devtools_app/lib/main.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Is this file a straight copy paste?

Reply:
yep straight copy and paste from the now stripped down file under web/main.dart